### ISSUE TITLE
Focus indicator for `va-select` reverted to browser defaults

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/vets-design-system-documentation#3282]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

The `va-select` component no longer seems to have a yellow focus indicator when in the `:focus` state. I'm not seeing any focus styles associated with the component except for the browser defaults.

![Select component with browser default focus styles.](https://github.com/user-attachments/assets/890cd9e1-c0ad-4b9e-a6ef-173ca521e612)

## What I expected to happen

The `va-select` component should have the same yellow focus outline that other components on VA.gov have. See screenshot of previous behavior, via @jeana-adhoc.

![Select component with yellow focus indicator.](https://github.com/user-attachments/assets/7041128c-bbbf-4eb2-b34f-f33867d64a31)

## Reproducing

- Formation version:
- Device: (e.g. iPhone 8, Macbook)
- Browser: (e.g. Firefox, IE 11)

Steps to reproduce:

1. Navigate to the [select component page](https://design.va.gov/components/form/select).
2. Press [tab] until focus moves to the select component example.
3. Observe that the focus state does not match previous styles.

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [x] Other

Noticed this while completing a staging review. This isn't blocking any work, but it's a degradation for anything using this component.

**Comment by caw310** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/3282?planning-poker) with Zenhub @Andrew565 @ataker @harshil1793 @it-harrison @jamigibbs @micahchiang @powellkerry @rsmithadhoc

**Comment by jamigibbs** [comment]
The mixin import for focus is:

```css
@import '../../mixins/focusable.css';
```
